packages/react/src/I18nProvider.tsx

```diff
@@ -11,13 +12,14 @@ export interface I18nProviderProps extends I18nContext {}
 
 const LinguiContext = React.createContext<I18nContext>(null)
 
+const config = getConfig({ configPath: process.env.LINGUI_CONFIG })
```

Comment:
This definitely won't work in browser because `getConfig` needs access to file system... You would need to turn it into a macro which loads config at compile time.